packages/components/src/dropdown-menu-v2-ariakit/stories/index.story.tsx

```diff
@@ -383,7 +450,13 @@ const Slot = () => {
 		[ dropdownMenuContext ]
 	);
 
-	return <ExampleSlotFill.Slot fillProps={ fillProps } bubblesVirtually />;
+	return (
+		<ExampleSlotFill.Slot
+			fillProps={ fillProps }
+			bubblesVirtually
+			style={ { display: 'contents' } }
```

Comment:
Adding `display: contents` to the `Slot` element is necessary, otherwise the slotfill example breaks the grid layout.